warp/src/main/java/com/schibsted/nmp/warp/components/WarpSpinner.kt

```diff
@@ -0,0 +1,44 @@
+package com.schibsted.nmp.warp.components
+
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.ProgressIndicatorDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.schibsted.nmp.warp.theme.WarpTheme
+
+@Composable
+fun WarpSpinner(
+    modifier: Modifier = Modifier,
+    size: WarpSpinnerSize = WarpSpinnerSize.Default,
+    strokeWidth: Dp = ProgressIndicatorDefaults.CircularStrokeWidth
```

Comment:
The strokewidth should be a predefined param to keep consistency throughout the design system across platforms. https://www.figma.com/design/oHBCzDdJxHQ6fmFLYWUltf/WARP---Components-2.0?m=auto&node-id=900-35603&t=381wrlkzoSir9LSh-1

Reply:
Lets remove this param from the public function